tools/build_commands

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
No. If people are okay with the switch to single-file command building, it can be removed. The rest of its functionality is replaced by `tools/install_commands`.